static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
```

Comment:
the copy of the profile we're currently editing we might not want to be a straight up copy of the profile anyway, i think it will eventually make sense to store the validation data in there too.